.github/workflows/static-checks.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
       TRAVIS_PULL_REQUEST_BRANCH: ${{ github.head_ref }}
       TRAVIS_PULL_REQUEST_SHA : ${{ github.event.pull_request.head.sha }}
       RUST_BACKTRACE: "1"
-      target_branch: ${TRAVIS_BRANCH}
```

Comment:
`TRAVIS_BRANCH` is set at line 12 to `github.base_ref`. At this point its value is not available?

Reply:
No, I changed it after it seemed to fail to evaluate it as environment variable, maybe it's possible with some special syntax but i couldn't find something like that.